src/exception/validation.ts

```diff
@@ -140,17 +140,7 @@ export const checkRequestedValue: Validator<ExceptionRecord> = ({ record, fieldN
   const validRequests: string[] = Object.values(ExceptionValue);
   const requestedExceptionValue = record.requested_exception_value;
 
-  if (requestedExceptionValue === undefined) {
-    return {
-      result: ValidationResultType.UNDEFINED,
-      message: `${fieldName} value is not defined`,
-    };
-  } else if (typeof requestedExceptionValue !== 'string') {
-    return {
-      result: ValidationResultType.TYPE_ERROR,
-      message: `${fieldName} value is not a string`,
-    };
-  } else if (!validRequests.includes(requestedExceptionValue)) {
+  if (requestedExceptionValue !== undefined && !validRequests.includes(requestedExceptionValue)) {
```

Comment:
Even before your changes, I am confused why this logic is needed... originally this property couldn't be undefined... If we restore the exception to require a string for `requested_exception_value` then we wouldn't need this `undefined` check.